DirectProgramming/DPC++FPGA/Tutorials/Features/scheduler_target_fmax/README.md

```diff
@@ -0,0 +1,166 @@
+# Scheduler Target FMAX
+This tutorial explains the `scheduler_target_fmax_mhz` attribute and its effect on the performance of Intel® FPGA kernels.
+
+***Documentation***:  The [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
+The [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit
+| What you will learn               |  The behavior of the `scheduler_target_fmax_mhz` attribute and when to use it on your kernel. <br> The effect this attribute can have on your kernel's performance on FPGA.
+| Time to complete                  | 15 minutes
+
+## Purpose
+This tutorial demonstrates how to use the `[[intel::scheduler_target_fmax_mhz(N)]]` attribute to direct the compiler to prioritize a high fMAX over a low II. If you are not yet familiar with the `[[intel::initiation_interval(N)]]` attribute which can change the initiation interval (II) of a loop to improve performance, refer to the prerequisite tutorial "Loop initiation_interval attribute".
```

Comment:
This paragraph should include how this attribute applies only to a single kernel.